Packaging.cmake

```diff
@@ -0,0 +1,64 @@
+if(NOT VERSION)
+	set(VERSION "unknown")
+endif()
```

Comment:
Oh, I think that `cmake` has some kind of variable scoping. Variables are only set locally (in `awesomeConfig.cmake`) and not in the calling "scope" (`CMakeLists.txt`). Never mind then, just leave this as-is.